sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/pipline/policies/BaseAppConfigurationPolicy.java

```diff
@@ -24,41 +27,67 @@
 
     public static final String USER_AGENT = String.format("%s/%s", StringUtils.replace(PACKAGE_NAME, " ", ""),
         BaseAppConfigurationPolicy.class.getPackage().getImplementationVersion());
-    
-    public static final String USER_AGENT_TYPE = "User-Agent";
 
     static Boolean watchRequests = false;
 
+    final Boolean isDev;
+
+    final Boolean isKeyVaultConfigured;
+    
+    public BaseAppConfigurationPolicy(Boolean isDev, Boolean isKeyVaultConfigured) {
+        this.isDev = isDev;
+        this.isKeyVaultConfigured = isKeyVaultConfigured;
+    }
+
     /**
+     * 
      * Checks if Azure App Configuration Tracing is disabled, and if not gets tracing information.
      *
      * @param request The http request that will be traced, used to check operation being run.
      * @return String of the value for the correlation-context header.
      */
-    private static String getTracingInfo(HttpRequest request) {
+    private String getTracingInfo(HttpRequest request) {
         String track = System.getenv(RequestTracingConstants.REQUEST_TRACING_DISABLED_ENVIRONMENT_VARIABLE.toString());
         if (track != null && track.equalsIgnoreCase("false")) {
             return "";
         }
-        String requestTypeValue = RequestType.WATCH.toString();
-        if (!watchRequests) {
-            requestTypeValue = request.getUrl().getPath().startsWith("/kv") ? RequestType.STARTUP.toString()
-                : RequestType.WATCH.toString();
-        }
-        if (requestTypeValue.equals(RequestType.WATCH.toString())) {
-            watchRequests = true;
-        }
+
+        String requestTypeValue = watchRequests ? RequestType.WATCH.toString() : RequestType.STARTUP.toString();
```

Comment:
Use the enum type here, and when performing a concatenation, the toString will be called automatically.